front/src/hooks/stylo-export.js

```diff
@@ -28,3 +28,17 @@ export default function useStyloExport (bibStyle) {
 
   return { exportFormats, exportStyles, exportStylesPreview, isLoading }
 }
```

Comment:
I cannot comment above that line but I think line 27 it should be `bibliography_style: bibStyle` now with the breaking change I introduced in image 0.0.9.